tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_gluster_heal_info.py

```diff
@@ -93,72 +90,57 @@ def get_volume_heal_info(vol):
         return ret_val
 
 
+def get_heal_info(volume, integration_id):
+    vol_heal_info = get_volume_heal_info(volume)
+    for brick_heal_info in vol_heal_info:
+        t_name = \
+            'clusters.%s.volumes.%s.nodes.%s.bricks.%s.split_brain_cnt'
+        ret_val[
+            t_name % (
+                integration_id,
+                volume['name'],
+                brick_heal_info['host_name'].replace('.', '_'),
+                brick_heal_info['brick_path'].replace('/', '|')
+            )
+        ] = brick_heal_info['split_brain_cnt']
+        t_name = \
+            'clusters.%s.volumes.%s.nodes.%s.bricks.%s.healed_cnt'
+        ret_val[
+            t_name % (
+                integration_id,
+                volume['name'],
+                brick_heal_info['host_name'].replace('.', '_'),
+                brick_heal_info['brick_path'].replace('/', '|')
+            )
+        ] = brick_heal_info['healed_cnt']
+        t_name = \
+            'clusters.%s.volumes.%s.nodes.%s.bricks.%s.heal_failed_cnt'
+        ret_val[
+            t_name % (
+                integration_id,
+                volume['name'],
+                brick_heal_info['host_name'].replace('.', '_'),
+                brick_heal_info['brick_path'].replace('/', '|')
+            )
+        ] = brick_heal_info['heal_failed_cnt']
+
+
 def get_metrics(CLUSTER_TOPOLOGY, CONFIG):
-    ret_val = {}
+    global ret_val
+    # threads = []
```

Comment:
remove the commented line

Reply:
No I have intentionally not removed because of reason in https://github.com/Tendrl/node-agent/pull/599/files#r136752747

Reply:
Ack